src/SamplesApp/UITests.Shared/Windows_UI_Xaml/Clipping/UIElement_Clip_Update.xaml.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Media;
+using Uno.UI.Samples.Controls;
+using Windows.Foundation;
+
+namespace UITests.Windows_UI_Xaml.Clipping;
+
+
+[Sample("Clipping", Description = """
+	1. Initially, there should be a button and a 100x100 border.
+	2. Click the button.
+	3. The border should become smaller, 80x80
+	4. Click the button once again
+	5. The border should become smaller, 50x50
+	""", IsManualTest = true)]
```

Comment:
This is a manual test due to a bug in RenderTargetBitmap where it shows the border as being clipped when it's not (when the InvalidateArrange is missing). We have no other good way to test this AFAIK.